[ERROR] TypeError: ctx.emit is not a function

I'm trying to use the theme but I keep getting this error. 

> » [ERROR] TypeError: ctx.emit is not a function
>     at tryLoadFile (C:\Home\Projects\Ui-Library\node_modules\sassdoc-theme-herman\node_modules\sassdoc-extras\src\description.js:26:11)     
>     at Function.description (C:\Home\Projects\Ui-Library\node_modules\sassdoc-theme-herman\node_modules\sassdoc-extras\src\description.js:15:21)
>     at module.exports (C:\Home\Projects\Ui-Library\node_modules\sassdoc-theme-herman\lib\prepareContext.js:151:10)
>     at Environment.herman [as theme] (C:\Home\Projects\Ui-Library\node_modules\sassdoc-theme-herman\index.js:22:3)
>     at theme (C:\Home\Projects\Ui-Library\node_modules\sassdoc\dist\sassdoc.js:351:23)
>     at _callee$ (C:\Home\Projects\Ui-Library\node_modules\sassdoc\dist\sassdoc.js:288:22)
>     at tryCatch (C:\Home\Projects\Ui-Library\node_modules\regenerator-runtime\runtime.js:62:40)
>     at Generator.invoke [as _invoke] (C:\Home\Projects\Ui-Library\node_modules\regenerator-runtime\runtime.js:296:22)
>     at prototype.<computed> [as next] (C:\Home\Projects\Ui-Library\node_modules\regenerator-runtime\runtime.js:114:21)
>     at step (C:\Home\Projects\Ui-Library\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
>     at C:\Home\Projects\Ui-Library\node_modules\babel-runtime\helpers\asyncToGenerator.js:28:13
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)



I tried to specify a specific file inside a test folder to try to identify the problem 

```json
"scripts": { 
    "documentsass": "sassdoc ./src/stylesheets/abstracts/test --theme herman"
}
```

sassdoc is installed locally not globally and it works without the --theme herman

This is the only file I have inside the folder _colors.scss

```scss
/*
    The main/primary color 
    → used for the links, the button background color, etc.  In general, it’s the main call-to-action color.
    The accent color 
    → used to highlight something important on the page. It shouldn’t be a variation of the primary color, but a complementary color.
    A scale of neutral colors 
    → It’s generally a scale of grayscale tones, to be used for text elements, subtle elements, borders, etc.
    Feedback colors → success, error, warning.
*/

// Theme Colors
/// Sass variables for defining theme colors and feedback colors.
/// @author Muhammad

/// @type variable - The main/primary color used for links, buttons, etc.
$primary: #0d6efd !default;
/// @type variable - The accent color for highlighting important elements.
$secondary: #6c757d !default;
/// @type variable - Color indicating success or positive feedback.
$success: #198754 !default;
/// @type variable - Color indicating informational messages or neutral feedback.
$info: #0dcaf0 !default;
/// @type variable - Color indicating a warning or cautionary feedback.
$warning: #ffc107 !default;
/// @type variable - Color indicating errors or critical feedback.
$danger: #dc3545 !default;
/// @type variable - A light neutral color for backgrounds or subtle elements.
$light: #f8f9fa !default;
/// @type variable - A dark neutral color for text or borders.
$dark: #212529 !default;
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 # Herman Changelog
 
+## 6.0.1: 2024-04-22
+
+- 🐛 BUGFIX: Do not overwrite `ctx.emit` method -
+  [#423](https://github.com/oddbird/sassdoc-theme-herman/issues/423)
+
 ## 6.0.0: 2024-03-01
 
 ### 🚀 New Features

--- a/lib/prepareContext.js
+++ b/lib/prepareContext.js
@@ -35,15 +35,19 @@ module.exports = (ctx) => {
 
   ctx.groups = ctx.groups || { undefined: 'General' };
 
-  // Apply default values for display.
+  // Apply default values.
   ctx.display = Object.assign({}, def.display, ctx.display);
-
-  if (!ctx.description) {
-    def.descriptionPath = './README.md';
+  /* istanbul ignore else */
+  if (!ctx.sort) {
+    ctx.sort = def.sort;
+  }
+  if (!ctx.herman) {
+    ctx.herman = def.herman;
+  }
+  if (!ctx.description && !ctx.descriptionPath) {
+    ctx.descriptionPath = './README.md';
   }
 
-  // Extend top-level context keys.
-  ctx = Object.assign({}, def, ctx); // eslint-disable-line no-param-reassign
   /* istanbul ignore else */
   if (!ctx.herman.sass?.sassOptions) {
     ctx.herman.sass = {

